Algorithms/Numeric/TemperatureConverter.cs

```diff
@@ -0,0 +1,90 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Algorithms.Numeric
+{
+    public static class Converter
+    {
+        static void Main(string[] args)
+        {
+
+            double C;
+            double K;
+            double F;
+
+            Console.WriteLine("If you want to convert from Celsius to Fahrenheit ... type 1 and press enter: ");
+            Console.WriteLine("If you want to convert from Fahrenheit to Celsius ... type 2 and hit enter:");
+            Console.WriteLine("If you want to convert from Fahrenheit to Kelvin ... type 3 and hit enter:");
```

Comment:
Codacy found an issue: [Remove this logging statement.](https://app.codacy.com/manual/siriak/C-Sharp/pullRequest?prid=6301588)